src/main/java/uk/nhs/digital/ps/migrator/model/hippo/BaseXlsReader.java

```diff
@@ -0,0 +1,40 @@
+package uk.nhs.digital.ps.migrator.model.hippo;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Iterator;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.xssf.usermodel.XSSFSheet;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+
+/**
+ * Shared functions for streaming and reading Excel sheets/rows.
+ */
+public class BaseXlsReader {
```

Comment:
As discussed, given how short-lived this project is going to be, I'm fine with leaving this as a base class rather than going the composition route.